src/main/java/com/kodilla/ecommercee/controller/UserController.java

```diff
@@ -1,38 +1,36 @@
 package com.kodilla.ecommercee.controller;
 
-import com.kodilla.ecommercee.domain.OrderDto;
-import com.kodilla.ecommercee.domain.ProductDto;
 import com.kodilla.ecommercee.domain.UserDto;
 import org.springframework.web.bind.annotation.*;
 
-import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.Random;
 
 import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
 
 @RestController
 @RequestMapping("/superShop")
 public class UserController {
-    private Random fourRandomNumbersGenerator = new Random();
-    private Integer a = fourRandomNumbersGenerator.nextInt(9);
-    private Integer b = fourRandomNumbersGenerator.nextInt(9);
-    private Integer c = fourRandomNumbersGenerator.nextInt(9);
-    private Integer d = fourRandomNumbersGenerator.nextInt(9);
+    private Random fiveRandomNumbersGenerator = new Random();
```

Comment:
Kontroler nie powinien realizować żadnej logiki. Powinno to być przeniesione do jakiejś klasy typu service, a kontroler powinien wywołać z niej odpowiednią usługę.

Reply:
@interceptor2048 Poprawione :)

Reply:
@interceptor2048 Mógłbyś proszę zobaczyć, czy jest już ok?